compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -10690,6 +10730,25 @@ private STNode parseRetryBody() {
         }
     }
 
+    private STNode parseOnFailClause() {
```

Comment:
Shall we add the grammar rule as a doc comment?

Reply:
Fixed with 3189bc6c6a9ee435ca57fa863f10d8e4f357ab6b

Reply:
Fixed with 3189bc6c6a9ee435ca57fa863f10d8e4f357ab6b